### PR TITLE
9.0 bringup

### PR DESCRIPTION
**Pull request by alviteri**

### CODE REVIEW

**Comment by MarijnS95** [comment]
@alviteri Response to your comment (https://github.com/sonyxperiadev/vendor-sony-oss-fingerprint/pull/28#issuecomment-414826058) in the right thread: Does this allow Suzu to boot and work somewhat properly? Because up till now I have never gotten it past the Sony logo, and do not have a UART attached to check what's wrong (pstore logs are completely scrambled making them incredibly difficult to read).

**Comment by alviteri** [comment]
Have you built with sonyaosp manifest ? If yes the problem can be gapps ... I have disabled them until opengapps upgrade to pie ....

**Comment by alviteri** [comment]
The manifest is now pure aosp

**Comment by MarijnS95** [comment]
It has been built without any modifications except some fixes to build errors (some of which have already been contributed to the Sony repos), and deployed on a completely clean/wiped device. I'll have to fetch all recent changes and try again sometime later, but I do not have the time to do so today or tomorrow.

**Comment by jerpelea** [comment]
please remove https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/d80ae153568d8cacbc2f8d86ba9d6f99b1e82066

**Comment by jerpelea** [comment]
some commits were cherry-picked from PR 
please rebase

**Comment by alviteri** [comment]
rebased

**Comment by jerpelea** [comment]
please remove 
 https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/8c36051ba180697140a724309189e9a1e272cd90

this commit has been reverted 
https://android.googlesource.com/device/google/wahoo/+/91d14832f0fceb574fca8a66684b6d61fdcb0044

**Comment by alviteri** [comment]
Then it was reverted again .... https://android.googlesource.com/device/google/wahoo/+/android-9.0.0_r1/manifest.xml#95

**Comment by jerpelea** [comment]
please rebase

**Comment by alviteri** [comment]
https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/767bedc3121f6b140adc84b9fef10fe9077d0d3b

hostpot wifi ok tested with vodafone IT

**Comment by jerpelea** [comment]
cherry-picked please rebase

**Comment by jerpelea** [comment]
please remove https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/29708ff21ae7179a459a7c875cabbb218625513f#diff-42093a6732b496e5669e16c7cf365289

it has been reordered

File: hardware/qcom/Android.mk [hunk 1]
```diff
@@ -23,9 +23,9 @@ audio-hal := hardware/qcom/audio
 bt-hal := hardware/qcom/bt/sdm845
 gps-hal := hardware/qcom/gps/sdm845
 display-hal := hardware/qcom/display/sde
-QCOM_MEDIA_ROOT := hardware/qcom/media/sdm845
+QCOM_MEDIA_ROOT := hardware/qcom/media/msm8998
 OMX_VIDEO_PATH := mm-video-v4l2
-media-hal := hardware/qcom/media/sdm845
+media-hal := hardware/qcom/media/msm8998
```
Comment (oshmoun) [inline hardware/qcom/Android.mk]: media-hal := ${QCOM_MEDIA_ROOT} please :)

Comment (alviteri) [reply hardware/qcom/Android.mk]: @oshmoun good idea doing in new commit

Comment (jerpelea) [reply hardware/qcom/Android.mk]: please squash https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/1547cfbe8976a0dd65e6e53584057572c23afc03

**Comment by jerpelea** [comment]
please upstream
SonyAosp/platform_hardware_qcom_media@4ef312b
SonyAosp/platform_hardware_qcom_media@84a11a6

**Comment by alviteri** [comment]
wait ifeq ($(SOMC_KERNEL_VERSION), 4.9) should be ifeq ($(TARGET_KERNEL_VERSION), 4.9) let me update

**Comment by alviteri** [comment]
we can't send the patch with sony needed (it should be all)

**Comment by alviteri** [comment]
but you asked me before to squash them 
now you are asking me to split them ......

**Comment by jerpelea** [comment]
1 patch switch mediaHAL to msm8998
1 patch define media-hal using QCOM_MEDIA_ROOT
from the new squashed commit

**Comment by jerpelea** [comment]
please rebase

**Comment by alviteri** [comment]
done

**Comment by jerpelea** [comment]
cherry-picked 
please rebase the commits

**Comment by jerpelea** [comment]
the sensors should run as a system service 
https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/adc965893cc7369353d58dddcdeb717d7299bf76

**Comment by alviteri** [comment]
have you build a new set of odm with system perms ?

**Comment by jerpelea** [comment]
please remove 
https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/0fbfd23815eb43ed6658fdbc6c226e207438f605

**Comment by jerpelea** [comment]
there is no reason for this change since our system image is big enough 
https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/9a99c6282ee485eb039c815a3e381d9f5ad216d6

**Comment by jerpelea** [comment]
there is no need to replace code with prebuilt binaries 
please remove 
https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/01cc92f5b48d4e4b679c7deac3a5801d8545e2ec

**Comment by alviteri** [comment]
please do https://github.com/SonyAosp/platform_hardware_qcom_bt/commits/android-9.0

**Comment by alviteri** [comment]
let me squash in one commit better

**Comment by alviteri** [comment]
https://github.com/SonyAosp/platform_hardware_qcom_bt/commit/8dd32c9dd856c8cb11ccf7462bf110b6666b75af

**Comment by jerpelea** [comment]
please remove 
https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/9204442e23e28216d85463e93fcd720909dfd94d

**Comment by jerpelea** [comment]
please remove the property set from the rc files 
the properties must be set on each device at build time 
https://github.com/sonyxperiadev/device-sony-common/pull/505/commits/de5704742dc7bb29c220d6aaf001d4db419d4482

**Pull request merged**